Coding Practice/Secure-Design-Principles.md

```diff
@@ -28,7 +28,7 @@ Example: By default, session timeouts are enabled. Users might be allowed to tur
 
 ### Principle of Least Privilege
 
-The principle of least privilege recommends that accounts have the least amount of privilege required to perform their business processes. This encompasses user rights, resource permissions such as CPU limits, memory, network, and file system permissions.
+The principle of least privilege recommends that accounts have the least amount of privilege required to perform their business processes. This encompasses user rights, resource permissions such as CPU (Central Processing Unit) limits, memory, network, and file system permissions.

```

Comment:
I appreciate breaking out all the acronyms - it's easy for us to get stuck in jargon and alphabet soup. However, I don't think CPU needs to be broken out, as it's common vernacular among most people at this point.

Reply:
Done